pillarbox-monitoring-grafana/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Dockerfile for grafana
-FROM grafana/grafana:11.0.0
+FROM grafana/grafana:10.4.1
```

Comment:
10.4.1 is the Grafana version used on AWS, correct?

Reply:
Currently yes.